src/main/java/org/cloudfoundry/promregator/lite/websecurity/SecurityConfig.java

```diff
@@ -0,0 +1,163 @@
+package org.cloudfoundry.promregator.lite.websecurity;
+
+//import org.cloudfoundry.promregator.config.InboundAuthorizationMode;
+//import org.cloudfoundry.promregator.endpoint.EndpointConstants;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.BeanIds;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.savedrequest.HttpSessionRequestCache;
+import org.springframework.security.web.savedrequest.RequestCache;
+
+import java.util.UUID;
+
+//@Configuration
+//@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
This can be deleted and was a leftover from when the `lite` codebase was a separate project